app/views/layouts/_navbar.html.erb

```diff
@@ -57,6 +57,15 @@
           </ul>
         </li>
 
+        <% if user_signed_in? %>
+          <li class="nav-item">
+            <%= link_to destroy_user_session_path, data: { turbo_method: :delete }, class: "nav-link" do %>
+              <i class="bi bi-box-arrow-left mobile-only"></i>
+              <%= t("navbar.sign_out") %>
+            <% end %>
+          </li>
+        <% end %>
+
```

Comment:
I searched for this button for ages in the desktop view 😅
I thought you had implemented it as the mock-up shows (a log-out btn in the right navbar side) and wondered if I had done something wrong with my setup.
I think we should do it the way the mock-up shows. At least that is my personal preference.
What do you think?

Reply:
Yeah this was admittedly a little sloppy. I have gone ahead and adapted the navbar to more closely match the mockup in 6b7c33e3a73b6a1b536b62b5582518185a9a6f70.